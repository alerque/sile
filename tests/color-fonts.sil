% KNOWNBAD https://github.com/sile-typesetter/sile/issues/1261
\begin[class=book,papersize=129mm x 198mm]{document}
\use[module=packages.color-fonts]
\bidi-on
\font[filename=tests/TestCLR-Regular.ttf]
a☺a

a☺🏻a

ع☺🏻ع
\end{document}
