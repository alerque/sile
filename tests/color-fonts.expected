Set paper size 	365.6692953	561.2598486
Begin page
Mx 	50.3506
My 	73.1061
Set font 	;10;400;;normal;;;LTR;tests/TestCLR-Regular.ttf
T	3	(a)
Push color	1	0.8784	0.5412
Mx 	60.3506
T	7	()
Pop color
Push color	0.898	0.6078	0.1451
T	8	()
Pop color
Push color	0.5176	0.2196	0.05098
T	9	(‚ò∫)
Pop color
Mx 	70.3506
T	3	(a)
Mx 	50.3506
My 	85.1061
T	3	(a)
Push color	0.9804	0.8784	0.8118
Mx 	60.3506
T	7	()
Pop color
Push color	0.898	0.6078	0.1451
T	8	()
Pop color
Push color	0.5176	0.2196	0.05098
T	9	(‚ò∫üèª)
Pop color
Mx 	70.3506
T	3	(a)
Mx 	50.3506
My 	97.1061
Set font 	;10;400;;normal;;RTL;tests/TestCLR-Regular.ttf
T	4	(ÿπ)
Pop color
Mx 	60.3506
T	9	(‚ò∫üèª)
Push color	0.5176	0.2196	0.05098
Pop color
Mx 	70.3506
T	8	()
Push color	0.898	0.6078	0.1451
Pop color
T	7	()
Push color	0.9804	0.8784	0.8118
T	4	(ÿπ)
Mx 	170.0669
My 	490.4464
Set font 	Gentium Plus;10;400;;normal;;;LTR
T	20	(1)
End page
Finish
